fdbserver/DataDistribution.actor.cpp

```diff
@@ -2912,18 +2932,26 @@ ACTOR Future<Void> waitForAllDataRemoved( Database cx, UID serverID, Version add
 	}
 }
 
-ACTOR Future<Void> storageServerFailureTracker(
-	DDTeamCollection* self,
-	TCServerInfo *server,
-	Database cx,
-	ServerStatus *status,
-	Version addedVersion )
-{
+ACTOR Future<bool> storageServerFailureTracker(DDTeamCollection* self, TCServerInfo* server, Database cx,
+                                               ServerStatus* status, Version addedVersion) {
 	state StorageServerInterface interf = server->lastKnownInterface;
 	loop {
-		state bool inHealthyZone = self->healthyZone.get().present() && interf.locality.zoneId() == self->healthyZone.get();
-		if(inHealthyZone) {
-			status->isFailed = false;
+		state bool inHealthyZone = false;
+		if (self->healthyZone.get().present()) {
+			if (interf.locality.zoneId() == self->healthyZone.get()) {
+				status->isFailed = false;
+				inHealthyZone = true;
+			} else if (self->healthyZone.get().get() == ignoreSSFailure) {
+				// Ignore all SS failures
+				status->isFailed = false;
+				status->isUndesired = false;
+				status->isWrongConfiguration = false;
+				TraceEvent("SSFailureTracker", self->distributorId)
```

Comment:
This message will be very spammy. Probably, we need to remove or comment out the message.

Reply:
Will do.